health-services/resource-generator/CHANGELOG.md

```
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.0.1 - 2025-01-30
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)